taxi_trips_duration_train_opt.ipynb

```
@@ -0,0 +1,756 @@
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from spark_esri import spark_start, spark_stop"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
```

Ruff (0.16.0)

[error] Undefined name `math`

(F821)